.github/workflows/container_scanning.yaml

```diff
@@ -0,0 +1,39 @@
+name: Docker container scanning
+
+on:
+  pull_request:
+  push:
+    branches: [main, radix]
+  workflow_dispatch:
+
+jobs:
+  vulnerability_scanning:
+    name: Trivy test
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v5
+
+      - name: Load openvds image
+        uses: "./.github/actions/load_openvds_image"
+
+      - name: Build docker image
+        run: |
+          docker build \
+          -f Dockerfile \
+          --build-arg OPENVDS_IMAGE=${{ env.OPENVDS_IMAGE_TAG }} \
+          --tag ${{ github.repository }}/docker-image:${{ github.sha }} \
```

Comment:
I recall we didn't use sha in snyk because for snyk different name was considered a different project and we would have a lot of garbage here.
There is no such problem for Trivy?

Reply:
Yes, you are correct. The image we build is only used within the workflow and therefore we can choose any name we want. I ended up using a name similar to what was used in the Trivy example of Trivy's repository.